src/worksheet.h

```diff
@@ -24,5 +24,29 @@ class Worksheet : public Napi::ObjectWrap<Worksheet> {
   Napi::Value WriteNumber(const Napi::CallbackInfo& info);
   Napi::Value WriteString(const Napi::CallbackInfo& info);
   Napi::Value WriteURL(const Napi::CallbackInfo& info);
+  Napi::Value Autofilter(const Napi::CallbackInfo& info);
+  Napi::Value DataValidationCell(const Napi::CallbackInfo& info);
+  Napi::Value DataValidationRange(const Napi::CallbackInfo& info);
   lxw_worksheet* worksheet = nullptr;
 };
+
+class DataValidation {
```

Comment:
Move this class to `data-validation.h`.

Reply:
This shouldn't be a C++ class. Remove from here, and just have a function in `worksheet.cc` that takes a `Napi::Object` and returns a `lxw_data_validation`. We don't need to do any copies of strings since `lxw_data_validation_cell` and `lxw_data_validation_range` will copy them anyway.